### PR TITLE
feat(CORSO-21): view all current orders and implement pagination

### PR DESCRIPTION
**Pull request by Mila5847**
JIRA: 
Ticket: CORSO-21
https://champlainsaintlambert.atlassian.net/browse/CORSO-21

## Context:
The ticket is about viewing all current orders through the admin panel.
Pagination was also implemented to facilitate viewing big amount of current orders.
## Changes
- Implementing Order Service
- Testing Order Service
- Implementing Pagination (mostly shown in the front-end)
- Add order tracking number to order entity.
### Before
- no page for current orders
### After
image
## Dev notes (Optional)
- Specific technical changes that should be noted
## Linked pull requests (Optional)
- pull request link

### CODE REVIEW

File: corso_website_fe/src/Components/OrderTile.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        color: black;
+    }
+
+    & .customer-full-name {
+        grid-area: 2 / 1 / 3 / 2; /* Place customer-full-name in the bottom-left grid cell */
+        padding-top: 30px;
+        color: black;
+    }
+
+    & .order-service {
```
Comment (DenisaNicoletaH) [inline corso_website_fe/src/Components/OrderTile.css:65]: What is the & supposed to be.

File: corso_website_springboot/src/test/java/com/corso/springboot/Order_Subdomain/datalayer/OrderRepositoryPersistenceTest.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    @BeforeEach
+    public void setup(){
+        orderRepository.deleteAll();
+        preSavedOrder = orderRepository.save(
+                new Order("982577e6-1909-46b8-8583-e08c9daa4e9b", "2023-01-01", "2023-02-01", Status.IN_PROGRESS, "Custom web development project", "John Doe", "john@example.com", "1234567890", "123 Main St")
+        );
+    }
+
+//    @Test
```
Comment (KarinaSofia) [inline corso_website_springboot/src/test/java/com/corso/springboot/Order_Subdomain/datalayer/OrderRepositoryPersistenceTest.java:26]: Same idea as my previous comment. If this test isn't necessary consider removing it. Good work tho on your persistence test!

**Review by KarinaSofia — APPROVED** [review]
Great implementation of get all current orders as well as pagination. Great work on testing too and beautiful implementation of the front end! Just consider removing any unnecessary commented out code from your files.

File: corso_website_fe/src/Components/Pagination.tsx
Comment (DenisaNicoletaH) [inline corso_website_fe/src/Components/Pagination.tsx:1]: I believe you should remove unecessary commented out code(in the other files too) in order to not fill them out and make them long for no reason.

**Review by DenisaNicoletaH — APPROVED** [review]
Other than what I left as comment, I have looked at your code and I haven't seen any issues. You have followed the naming convention and the code is well done.

File: corso_website_fe/src/Components/OrderTile.css (hunk 1 above)
Comment (Mila5847) [reply corso_website_fe/src/Components/OrderTile.css:65]: The & represents the parent selector of a block.  So if we put & .order-service it means we are targeting the nested elements inside the parent selector

File: corso_website_springboot/src/test/java/com/corso/springboot/Order_Subdomain/datalayer/OrderRepositoryPersistenceTest.java (hunk 2 above)
Comment (Mila5847) [reply corso_website_springboot/src/test/java/com/corso/springboot/Order_Subdomain/datalayer/OrderRepositoryPersistenceTest.java:26]: yep, they will be removed!

File: corso_website_fe/src/Components/Pagination.tsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        </div>
+
+    );
+}
+
+export default CustomPagination;
+
+
+
+// import * as React from 'react';
```
[the remark this answers is not in the document]
Comment (Mila5847) [reply corso_website_fe/src/Components/Pagination.tsx:59]: It was an accident. Thank you for bringing it up! It will be fixed!

**Pull request merged**